UnitsNet.Tests/CustomCode/MolarityTests.cs

```diff
@@ -30,29 +30,104 @@ public class MolarityTests : MolarityTestsBase
     {
         protected override double CentimolesPerLiterInOneMolesPerCubicMeter => 1e-1;
         protected override double DecimolesPerLiterInOneMolesPerCubicMeter => 1e-2;
-        protected override double MicromolesPerLiterInOneMolesPerCubicMeter => 1e3;
-        protected override double MillimolesPerLiterInOneMolesPerCubicMeter => 1;
         protected override double MolesPerLiterInOneMolesPerCubicMeter => 1e-3;
+        protected override double MillimolesPerLiterInOneMolesPerCubicMeter => 1;
+        protected override double MolesPerCubicMeterInOneMolesPerCubicMeter => 1;
+        protected override double MicromolesPerLiterInOneMolesPerCubicMeter => 1e3;
         protected override double NanomolesPerLiterInOneMolesPerCubicMeter => 1e6;
         protected override double PicomolesPerLiterInOneMolesPerCubicMeter => 1e9;
-        protected override double MolesPerCubicMeterInOneMolesPerCubicMeter => 1;
+
+        protected override double MolarInOneMolesPerCubicMeter => 1E-3;
+        protected override double MillimolarInOneMolesPerCubicMeter => 1;
+        protected override double MicromolarInOneMolesPerCubicMeter => 1E3;
+        protected override double NanomolarInOneMolesPerCubicMeter => 1E6;
+        protected override double PicomolarInOneMolesPerCubicMeter => 1E9;
+
+        private static double MolarMassHClInGramsPerMole = 36.46;
+        private static double MassOfSubstanceInGrams = 5;
```

Comment:
If this is an arbitrarily chosen value for a particular test case/scenario then I recommend inlining the value directly into the test methods so it's easier to read each test method on its own.

If the value is well defined or well known, then it should be renamed to something descriptive.

If I am reading it right, only a handful of these fields are well defined constants:
- MolarMassHClInGramsPerMole 
- MolarMassOfEthanolInGramsPerMole 
- DensityOfEthanolInKgPerCubicMeter 
- VolumeConcentration_0_5M_Ethanol 

The rest of them I suggest to be inlined into the test methods, so you can read the actual test case numbers for each test method. That makes it easier to read, in my opinion.